### PR TITLE
update quickstart readme, tests, and file names

### PR DESCRIPTION
**Pull request by victoriaxyz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,88 @@
...
-## Testing
+- Install `@clerk/nextjs`
+- Set your Clerk API keys
+- Add Clerk's middleware
+- Add `<ClerkProvider />` and Clerk components
+- Run E2E Cypress tests
 
-1. Create a test account using [Clerk Dashboard](https://dashboard.clerk.com) or by signing up on the application.
-2. Copy the `cypress.env.example.json` file to `cypress.env.json`.
+## Deploy
```
Comment (anagstef) [inline README.md]: ❓ Do we need instructions for `Deploy` for this repo?

File: README.md [hunk 2]
```diff
@@ -1,33 +1,88 @@
...
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fclerk%2Fclerk-nextjs-app-quickstart&env=NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY,CLERK_SECRET_KEY&envDescription=Clerk%20API%20keys&envLink=https%3A%2F%2Fclerk.com%2Fdocs%2Fquickstart%2Fnextjs&redirect-url=https%3A%2F%2Fclerk.com%2Fdocs%2Fquickstart%2Fnextjs)
 
-3. Set `test_user` and `test_password` to the credentials of the test account.
-4. Open Cypress while keeps the development server running:
+## Running the template
 
 ```bash
-npm run cypress:open
+git clone https://github.com/clerk/clerk-nextjs-app-quickstart
```
Comment (anagstef) [inline README.md]: ❓  Why do we suggest to run another template here?

Comment (victoriaxyz) [reply README.md]: Oops, this was copied over from the other README.

Comment (victoriaxyz) [inline README.md]:
```suggestion
npm run cypress:open
```

File: README.md (hunk 1 above)
Comment (victoriaxyz) [reply README.md]: This step is included in the default READMEs but we can remove it if you think this doesn't add any value.

Comment (anagstef) [reply README.md]: Removed it because it may be misleading!

**Review by anagstef — APPROVED** [review]

**Pull request merged**